src/miner_lora.erl

```diff
@@ -842,7 +842,11 @@ packet_rssi(Packet, UseRSSIS) ->
                 false ->
                     %% Just use RSSIC.
                     fun (Obj) ->
-                        maps:get(<<"rssic">>, Obj, undefined)
+                        SNR = maps:get(<<"lsnr">>, Obj, 0),
+                        case SNR < 0 of
+                            true -> SNR + maps:get(<<"rssic">>, Obj, undefined);
```

Comment:
Couldn’t this line result in an arithmetic operation error if rssic is undefined?

Reply:
Looks like it would, yes. I think a missing `rssic` is a brutal fatal error, though. So the arithmetic error is appropriate to propagate?

Reply:
it's defined to be present in this protocol version